src/init/init.ts

```diff
@@ -172,9 +172,10 @@ export function runGenerator(generatorName, options): Promise<any> {
   options = options || {};
   let templateName = options.templateName || generatorName;
 
-  return new Promise((resolve, reject) => {
-    resolve(options.env || createYeomanEnvironment());
-  }).then(function(env: YeomanEnvironment) {
+  return ((options.env)
```

Comment:
extra parens here made this a little harder for me to read, I initially wanted to parse it as an arrow function